### PR TITLE
send trusted advisor events to slack

### PR DESCRIPTION
**Pull request by wshands**
Addresses https://ucsc-cgl.atlassian.net/browse/SEAB-2224

### CODE REVIEW

File: cloud-watch-to-slack-testing-58006136-c30b-43d1-880c-7b36f860e7c8/deployment/index.js [hunk 1]
```diff
@@ -153,6 +153,17 @@ function processEvent(event, callback) {
     const newState = message.detail.newEvaluationResult.complianceType;
     messageText = `${alarmName} state is now ${newState}`;
     sendMessageToSlack(messageText, callback);
+  } else if (message.source == "aws.trustedadvisor") {
```
Comment (garyluu) [inline cloud-watch-to-slack-testing-58006136-c30b-43d1-880c-7b36f860e7c8/deployment/index.js:156]: `===`

**Review by garyluu — APPROVED** [review]

**Review by denis-yuen — APPROVED** [review]

**Review by Richard-Hansen — APPROVED** [review]

File: cloud-watch-to-slack-testing-58006136-c30b-43d1-880c-7b36f860e7c8/deployment/index.js (hunk 1 above)
Comment (wshands) [reply cloud-watch-to-slack-testing-58006136-c30b-43d1-880c-7b36f860e7c8/deployment/index.js]: fixed

File: cloud-watch-to-slack-testing-58006136-c30b-43d1-880c-7b36f860e7c8/deployment/index.js [hunk 2]
```diff
@@ -148,12 +148,23 @@ function processEvent(event, callback) {
...
 
-  if (message.source == "aws.config") {
+  if (message.source === "aws.config") {
     const alarmName = message.detail.configRuleName;
     const newState = message.detail.newEvaluationResult.complianceType;
     messageText = `${alarmName} state is now ${newState}`;
     sendMessageToSlack(messageText, callback);
-  } else if (message.source == "aws.guardduty") {
+  } else if (message.source === "aws.trustedadvisor") {
+    const detail_type = message["detail-type"];
```
Comment (coverbeck) [inline cloud-watch-to-slack-testing-58006136-c30b-43d1-880c-7b36f860e7c8/deployment/index.js:157]: Not worth changing, and we should just have this in the linter to enforce it, but we should be using camel case, not snake case, for variable names in JavaScript.

**Review by coverbeck — APPROVED** [review]

File: cloud-watch-to-slack-testing-58006136-c30b-43d1-880c-7b36f860e7c8/deployment/index.js (hunk 2 above)
Comment (wshands) [reply cloud-watch-to-slack-testing-58006136-c30b-43d1-880c-7b36f860e7c8/deployment/index.js]: fixed

**Pull request merged**